projects/kubernetes-sigs/image-builder/buildspecs/build-1-20-ubuntu-raw.yml

```diff
@@ -0,0 +1,17 @@
+version: 0.2
+
+env:
+  secrets-manager:
+    AMI_ID: "raw-image-build:ami-id"
+    KEY_NAME: "raw-image-build:key-name"
+    INSTANCE_TYPE: "raw-image-build:instance-type"
```

Comment:
I'm not at all against this, but do they have anything to be a secret. I was thinking, if it these were from parameter store/something else, might it be easier to manage with CDK. ex. we can control the instance type from CDK as a config. Please feel free to disagree.

Reply:
Yeah I was in two minds about this. At first I thought the AMI ID should be hidden from view, but I guess we already specify AMIs in our public Prowjobs repo and stuff. And they're private AMIs not shared with anyone. So putting them in CDK as an env var should be good enough